test/append_test.py

```diff
@@ -15,6 +15,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from builtins import range
```

Comment:
unused?

Reply:
`futurize` did that.  Though maybe it isn't needed.  I'll double check.